src/StatsdClient/StatsdBuilder.cs

```diff
@@ -46,20 +46,27 @@ public StatsdData BuildStatsData(StatsdConfig config)
             return new StatsdData(metricsSender, statsBufferize, transport, telemetry);
         }
 
-        private static string GetServerName(StatsdConfig config)
+        private static DogStatsdEndPoint BuildEndPoint(StatsdConfig config)
         {
             var statsdServerName = !string.IsNullOrEmpty(config.StatsdServerName)
                             ? config.StatsdServerName
                             : Environment.GetEnvironmentVariable(StatsdConfig.DD_AGENT_HOST_ENV_VAR);
 
-            if (string.IsNullOrEmpty(statsdServerName))
+            var pipeName = config.PipeName;
```

Comment:
Would we not want to ever set the pipe name with a config variable?

Reply:
Env variable added.